### PR TITLE
Update navbar and add additional sub-links

### LINKED ISSUE #94: Update the header: Add sub-links to roster and batch-upload pages
# [Designs](https://share.goabstract.com/f628fb10-aaa3-45d4-af7b-c2c255e44cc6)
• Rename title to ECE Reporter:
• When logged in, add sub-links for roster and batch-upload:
Screen Shot 2020-09-18 at 8 39 22 AM

• Clicking the logo and ECE reporter section should take the user back to the getting started page.

### PR DESCRIPTION
**Pull request by JordanGuinn**
Closes #94.  Associated with https://github.com/ctoec/component-library/pull/72.

### CODE REVIEW

File: client/src/containers/App/App.tsx [hunk 1]
```diff
@@ -17,11 +17,21 @@ const App: React.FC = () => {
...
         navItems={[
+          {
+            type: 'primary',
+            text: 'Roster',
+            path: '/roster',
+          },
+          {
+            type: 'primary',
+            text: 'Batch upload',
+            path: '/success', //  TODO: Actually route to batch upload when it's available
```
Comment (jlhogan) [inline client/src/containers/App/App.tsx:33]: batch upload is just `/upload` :)

**Review by jlhogan — APPROVED** [review]

File: client/src/containers/App/App.tsx [hunk 2]
```diff
@@ -52,6 +62,11 @@ const App: React.FC = () => {
         ]}
         userFirstName={user?.firstName}
       />
+      <div className="usa-nav__secondary usa-nav__secondary--extended">
+        <ul className="usa-nav__secondary-links">
```
Comment (mmazanec22) [inline client/src/containers/App/App.tsx:66]: Logout is already handled by the header-- why add this here?  Also we can't have a link with no path, it won't be able to receive keyboard focus.  And if something is part of the nav it needs to be contained within the nav element.

Comment (JordanGuinn) [reply client/src/containers/App/App.tsx:66]: Oh whoops - I didn't even realize I left this in here (was from me experimenting with stuff)

File: client/src/containers/App/App.tsx (hunk 1 above)
Comment (JordanGuinn) [reply client/src/containers/App/App.tsx:33]: Well THATS easy enough

**Review by mmazanec22 — APPROVED** [review]

**Review by bamader — APPROVED** [review]
LGTM!

**Review by kmagida110 — APPROVED** [review]
Looks good, good intro for me to the component library

**Pull request merged**